graphviz/jupyter_integration.py

```diff
@@ -1,12 +1,70 @@
 """Display rendered graph as SVG in Jupyter Notebooks and QtConsole."""
-
 from . import piping
 
 __all__ = ['JupyterSvgIntegration']
 
+_default_jupyter_representation = 'image/svg+xml'
+
+_jupyter_repr_functions = {
+    'image/svg+xml': lambda graph: graph.pipe(format='svg',
+                                              encoding=graph._encoding),
+    'image/png': lambda graph: graph.pipe(format='png'),
+    'image/jpeg': lambda graph: graph.pipe(format='jpeg'),
+
+    # The following does not work yet. Not clear why, because the documentation
+    # has only few information about them.
+    # 'application/json': lambda graph: json.loads(
+    #     graph.pipe(format='json', encoding=graph._encoding)),
+    # 'application/pdf': lambda graph: graph.pipe(format='pdf'),
+
+    # No reasonable representation
+    # 'text/plain': None,
+    # 'text/latex': None,
+    # 'text/html': None,
+    # 'application/javascript': None,
+    # 'text/markdown': None,
+}
+
+JUPYTER_REPRESENTATIONS = set(_jupyter_repr_functions.keys())
+
+
+def set_default_jupyter_representation(jupyter_representation: str) -> str:
+    """Change the default jupyter_representation"""
+    global _default_jupyter_representation
+
+    if jupyter_representation not in JUPYTER_REPRESENTATIONS:
+        similar = {
+            r
+            for r in JUPYTER_REPRESENTATIONS
+            if jupyter_representation.lower() in r}
+        if similar:
+            raise ValueError(
+                f'unknown jupyter representation: {jupyter_representation!r}\n'
+                f'Suggestions: {similar}')
+        else:
+            raise ValueError(
+                f'unknown jupyter representation: {jupyter_representation!r}')
+
+    old = jupyter_representation
+    _default_jupyter_representation = jupyter_representation
+    return old
+
 
 class JupyterSvgIntegration(piping.Pipe):
     """Display rendered graph as SVG in Jupyter Notebooks and QtConsole."""
 
-    def _repr_svg_(self):
-        return self.pipe(format='svg', encoding=self._encoding)
+    # def _repr_svg_(self):
+    #     return self.pipe(format='svg', encoding=self._encoding)
+
+    def _repr_mimebundle_(self, include=None, exclude=None, **kwargs):
```

Comment:
nit: Let's `del` kwargs in the first line explicitly to make explicit we intentionally ignore it (or use `**_):`).